cpp/src/gandiva/precompiled/extended_math_ops.cc

```diff
@@ -212,6 +239,24 @@ ENUMERIC_TYPES_UNARY(RADIANS, float64)
   }
 ENUMERIC_TYPES_UNARY(DEGREES, float64)
 
+// Ceil
+#define CEIL(IN_TYPE)                                                      \
+  FORCE_INLINE                                                             \
+  gdv_##IN_TYPE ceil_##IN_TYPE(gdv_##IN_TYPE in) {                         \
+    return static_cast<gdv_##IN_TYPE>(ceil(static_cast<long double>(in))); \
+  }
+CEIL(float32)
+CEIL(float64)
+
+// Ceil
```

Comment:
I think this comment was wrong, please fix it.

```suggestion
// Floor```

Reply:
Solved